stl/inc/atomic

```diff
@@ -1843,6 +2045,47 @@ _Ty atomic_fetch_xor_explicit(
     return _Mem->fetch_xor(_Value, _Order);
 }
 
+template <class _Ty>
+void atomic_wait(const volatile atomic<_Ty>* _Mem, typename atomic<_Ty>::value_type _Expected) {

```

Comment:
We should strengthen `noexcept` on all of these non-member functions:
```suggestion
void atomic_wait(const volatile atomic<_Ty>* _Mem, typename atomic<_Ty>::value_type _Expected) noexcept /* strengthened */ {
```

Reply:
Apparently it is a bug in proposal:
https://github.com/brycelelbach/wg21_p1135_cpp20_synchronization_library/issues/13
So it will become standard defect...

Reply:
addressed